tests/modules/programs/yazi/settings.nix

```
@@ -5,35 +5,35 @@
     enable = true;
 
     keymap = {
-      input.keymap = [
+      input.keymap_prepend = [
```

There is still some `keymap_prepend` left

```suggestion
      input.prepend_keymap = [
```